topics/single-cell/tutorials/scrna-data-formats/slides.html

```diff
@@ -0,0 +1,578 @@
+
+It's important to note that AnnData does not support sparse matrices, therefore large single-cell files with many expression values of zero will not be very efficient to store in memory.
+
+**Supported languages:** Python
+
+**Supported packages:** anndata
+
+---
+
+### Loom
+
+.pull-left[
+
+- Based on h5ad format
+
+- Efficient for large datasets
+
+- Supports sparse matrices and hierarchical data
+
+- Graph objects for graphs and clusters
+
+<br/><br/>
+
+- **File extensions:**
+  - .loom
+
+]
+
+.pull-right[
+
+![Basic structure of single-cell data](../../images/scrna-data-formats/gene-expression-diagram.png)
+
+]
+
+???
+
+Loom is another format based on HDF5. Loom objects are supported in various different programming languages and supports sparse matrices making it efficient for large data files.
+
+**Supported languages:** 
+- Python
+- R
+- MATLAB
+- Mathematica
+- C
+- C++
+- Java
+- Ruby
+
+**Supported packages:**
+- LoomPy (Python)
+- LoomR (R)
+
+---
+
+### Seurat
+
+.pull-left[
+
+- R-based format used by the Seurat package
+
+- Allows for easy integration with the suite of Seurat tools
+
+<br/><br/>
+
+- **File extensions:**
+  - .rds
+  - .rda
+  - .rdata
+
+]
+
+.pull-right[
+
+![Basic structure of single-cell data](../../images/scrna-data-formats/gene-expression-diagram.png)
+
+]
+
+???
+
+Seurat is an R-based format that is commonly used with the Seurat package (the naming convention is a bit confusing!). This is a software package that contains various processing and analysis tools for single-cell data.
+
+**Supported languages:** R
+
+**Supported packages:** seurat
+
+---
+
+### Single Cell Experiment (SCE)
+
+.pull-left[
+
+- R-based format provided by the Bioconductor package SingleCellExperiment
+
+- Integrates well with the Bioconductor ecosystem
+
+<br/><br/>
+
+- **File extensions:**
+  - .rds
+  - .rda
+  - .rdata
+
+]
+
+
+.pull-right[
+
+![Basic structure of single-cell data](../../images/scrna-data-formats/gene-expression-diagram.png)
+
+]
+
+???
+
+Single Cell Experiment (SCE) is another R-based format that is widely used within the Bioconductor ecosystem of tools for both processing and analysis of single-cell data.
+
+**Supported languages:** R
+
+**Supported packages:** SingleCellExperiment
+
+---
+
+### CellDataSet (CDS)
+
+.pull-left[
+
+- R-based format used by the Monocle package
+
+- Integrates well with the Monocle package for various types of analysis
+
+<br/><br/>
+
+- **File extensions:**
+  - .rds
+  - .rda
+  - .rdata
+
+]
+
+.pull-right[
+
+![Basic structure of single-cell data](../../images/scrna-data-formats/gene-expression-diagram.png)
+
+]
+
+???
+
+CellDataSet (CDS) is the last of the R-based formats that is commonly used within the Monocle package which contains tools for performing various types of analysis.
+
+**Supported languages:** R
+
+**Supported packages:** monocle
+
+---
+
+### Compressed Files

```

Comment:
This slide does not seem to fit here. Normally compressed files are downloaded from public repos. This slide can go after Data sources slide.

Reply:
Yeah that makes sense, I thought it was an important slide to add but wasn't too sure where it would best fit in. I'll move to the end!